presto-product-tests/conf/presto/etc/catalog/hive.properties

```diff
@@ -7,7 +7,7 @@
 
 connector.name=hive-cdh5
 hive.metastore.uri=thrift://hadoop-master:9083
-hive.metastore.thrift.client.socks-proxy=hadoop-master:1080
+hive.metastore.thrift.client.socks-proxy=hadoop-master:1180
```

Comment:
I've just built and published new images. I've just run tests against new docker image. Will start a PR once they pass so you can piggy back on that in your PR or take mine instead.